source/reference/limitations.txt

```diff
@@ -32,8 +32,8 @@ General Limitations
   6.2. Only major MongoDB releases such as 6.0 or 7.0. are supported.
   For more information on MongoDB versioning, see
   :ref:`release-version-numbers`.
-- The source and destination clusters must have the same release
-  version.
+- The source and destination clusters must have the same major and minor
+  release version, but can have different patch releases.
```

Comment:
[suggestion]

I think it might be helpful to add a couple sub-bullets outlining supported and unsupported syncronizations. As in:

- You can sync between a cluster running MongoDB 6.0.4 and MongoDB 6.0.9.
- You cannot sync between a cluster running MongoDB 5.0.12 and MongoDB 6.0.12.

Reply:
Fixed